src/TruncatedTooltip/TruncatedTooltip.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Link, Typography } from "@mui/material";
+import { Meta, StoryFn, StoryObj } from "@storybook/react";
+import React from "react";
+import TruncatedTooltip from "./TruncatedTooltip";
+import { TruncatedTooltipProps } from "./TruncatedTooltip.types";
+
+export default {
+  component: TruncatedTooltip,
+  title: "General/TruncatedTooltip"
+} satisfies Meta<typeof TruncatedTooltip>;
+
+const Template: StoryFn<TruncatedTooltipProps> = args => {
+  return (
+    <Box
+      sx={{
+        width: "220px"
+      }}
+    >
+      <TruncatedTooltip {...args} />
+    </Box>
+  );
+};
+
+const TypographyTemplate: StoryFn<TruncatedTooltipProps> = args => {
+  return (
+    <Box
+      sx={{
+        width: "220px"
+      }}
+    >
+      <TruncatedTooltip component={Typography} {...args} />
+    </Box>
+  );
+};
+
+// const LinkTemplate: StoryFn<TruncatedTooltipProps> = args => {};
+
+export const Default: StoryObj<typeof TruncatedTooltip> = {
+  args: {
+    children: "This is a long text that will be truncated",
+    href: "This/can/take/an/href/prop"
+  },
+
+  render: TypographyTemplate
+};
```

Comment:
Is this not more correct, and removes the additional template?

```suggestion
export const Default: StoryObj<typeof TruncatedTooltip> = {
  args: {
    children: "This is a long text that will be truncated",
    component: Typography,
    href: "This/can/take/an/href/prop"
  },

  render: Template
};
```